mffpy/writer.py

```diff
@@ -43,18 +43,16 @@ def __init__(self, filename: str, overwrite: bool = False):
     def create_directory(self):
         """Creates the directory for the recording."""
         if not self.file_created:
-            exist_ok = self.overwrite
-            makedirs(self.mffdir, exist_ok=exist_ok)
+            if self.overwrite:
+                rmtree(self.mffdir)
```

Comment:
This results in a `FileNotFoundError` if there is no MFF directory to overwrite. If overwrite is True, the writing should be successful whether there is something to overwrite or not. So let's check that the directory exists before running `rmtree`.